packages/core/src/process/transform/convertDeploymentVertexToExecutionVertex.ts

```diff
@@ -202,7 +202,9 @@ async function convertArgs(
   const resolvedArgs = [];
 
   for (const arg of args) {
-    const resolvedArg = await resolveParameter(arg, transformContext);
+    const resolvedArg = isBytesArg(arg)
```

Comment:
I was going to suggest creating a wrapping function here, but it would end up called `resolved` or something equally awful. I think we do nothing, wait for another case then abstract.